content/en/news/releases/1.7.x/announcing-1.7/change-notes/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Change Notes
+description: Istio 1.7 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- **Added** config option `values.global.proxy.holdApplicationUntilProxyStarts`,
+which causes the sidecar injector to inject the sidecar at the start of the
+pod's container list and configures it to block the start of all other
+containers until the proxy is ready.  This option is disabled by default.
+ ([Issue #11130](https://github.com/istio/istio/issues/11130))
+
+## Security
+
+- **Added** Enabled the workload cert rotate automatically in VM
+ ([Issue #24554](https://github.com/istio/istio/issues/24554))
+
+- **Added** support for creation of CSRs using ECC based certificates.
+ ([Issue #23226](https://github.com/istio/istio/issues/23226))
+
+- **Fixed** an issue preventing the use of source principal based authorization at Istio Gateway when the Server's TLS mode is `ISTIO_MUTUAL`.
+
```

Comment:
```suggestion

-  **Added**  SDS support for Client Certificate and CA certificate used for TLS/mTLS Origination from Egress 
    Gateway.
  ([Issue #14039](https://github.com/istio/istio/issues/14039))
  
```

Reply:
This is a good one to add. Can you link istio.io doc if we have one?